src/config.rs

```diff
@@ -0,0 +1,37 @@
+use std::io::Read;
+
+use once_cell::sync::OnceCell;
+
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+pub struct StoreConfig {
```

Comment:
I think we should keep config close to where it is used and then import it into src/config.rs